examples/hybrid/sphere/deformation_flow.jl

```diff
@@ -219,8 +224,21 @@ function rhs!(dydt, y, parameters, t, alpha, beta)
     cw = If2c.(w)
     cuvw = Geometry.Covariant123Vector.(uₕ) .+ Geometry.Covariant123Vector.(cw)
 
+    corrected_antidiff_flux = similar(dρq1)
     @. dρq1 = beta * dρq1 - alpha * hdiv(cuvw * ρq1) + alpha * ystar.ρq1
-    @. dρq1 -= alpha * vdivf2c(Ic2f(ρ) * third_order_upwind_c2f.(w, ρq1 ./ ρ))
+    @. corrected_antidiff_flux = vdivf2c(
+        Ic2f(ρ) *
+        C *
+        (
+            third_order_upwind_c2f.(w, ρq1 ./ ρ) -
+            first_order_upwind_c2f.(w, ρq1 ./ ρ)
+        ),
+    )
+    @. dρq1 -=
+        alpha * (
+            vdivf2c(Ic2f(ρ) * first_order_upwind_c2f.(w, ρq1 ./ ρ)) +
```

Comment:
```suggestion
            vdivf2c(Ic2f(ρ) * first_order_upwind_c2f(w, ρq1 / ρ)) +
```

Reply:
Oh, thanks for spotting those Charlie! Very good eyes :eyes: